plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -200,59 +195,49 @@ public String toString() {
 		return IAuthorizationContext.AUTHSCHEME_OAUTH;
 	}
 	
-	private class AuthLinkHandler extends SelectionAdapter{
-		private String link = "";
-		private Shell shell;
-
-		AuthLinkHandler(Shell shell, IAuthorizationDetails details){
-			this.shell = shell;
-			if(details != null) {
```

Comment:
this was moved to #createControls, i dont understand what this is all about, it appears to be "null" in all my tests